regression/strings-smoke-tests/performance.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+'''
+This script collects the running times of the tests present in the current
+directory.
+
+It does not run the tests but reads the already present output files.
+
+Usage:
+    python performance.py
+
+Dependencies:
+    gnuplot http://www.gnuplot.info/
+'''
+
+import time
+from subprocess import check_output
+from subprocess import check_call
+
+git_output = check_output(['git', 'show', 'HEAD'])
+commit = git_output.split('\n')[0]
+commit_id = time.strftime('%Y_%m_%d__%H_%M_%S')+'__'+commit[7:]
+
+process = check_output(['grep', '^Runtime decision procedure', '-R'])
```

Comment:
@smowton As suggested above, this doesn't work because we cannot write the output of a grep in the directory we are currently grepping. I get the following error:
```
writing to file performance_2017_06_05__14_53_24__56800d52eca33b06978f8863f333ef43fbabc6c3.out
grep: input file 'performance_2017_06_05__14_53_24__56800d52eca33b06978f8863f333ef43fbabc6c3.out’ is also the output
Traceback (most recent call last):
  File "performance.py", line 29, in <module>
    check_call(['grep', '^Runtime decision procedure', '-R'], stdout=f)
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['grep', '^Runtime decision procedure', '-R']' returned non-zero exit status 2
```